WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/Downloadable Files/ProductDownloadFileViewModelTests.swift

```diff
@@ -0,0 +1,297 @@
+import XCTest
+@testable import WooCommerce
+@testable import Yosemite
+
+final class ProductDownloadFileViewModelTests: XCTestCase {
```

Comment:
Anyway, these tests look good to me, but probably they need to be updated after the previous changes.

Reply:
right. updated